HW1/P2/P2.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import matplotlib.pyplot as plt 
 import matplotlib.cm as cm
+import random
```

Comment:
Dear Ray / cs205 staff--

I just noticed that this "P2.py" file should have been omitted from the final pull request. It was ultimately replaced by the two files below: "P2a.py" and "P2b.py" as were requested in the spec. If possible, please ignore this file as it was a remnant of an early commit that was never removed (sorry for the oversight!). 

Thanks,
Andrew